src/renderer/components/subscriptions-tab-ui/subscriptions-tab-ui.js

```diff
@@ -51,6 +55,10 @@ export default defineComponent({
     }
   },
   computed: {
+    currentLocale: function () {
+      return this.$i18n.locale.replace('_', '-')
+    },
+
```

Comment:
Looks like this is unused? I'm guessing it's leftover from testing?
```suggestion
```

Reply:
Yes, thank you - as for the first one, that was a relic from before that also wasn't being used as far as I could see, but yeah, definitely need to interrogate those more rather than taking an overly cautious stance